src/components/canvas/CurrentColor.js

```diff
@@ -13,7 +13,7 @@ const styles = {
 
 class CurrentColor extends Component {
   render() {
-    let newStyle = Object.assign({}, styles);
+    let newStyle = {...styles};
```

Comment:
This can be a `const`, and you can move the `backgroundColor into the object literal. I like how you did the spread with `...styles` and got rid of `Object.assign`.